crypto/s2n_composite_cipher_aes_sha.c

```diff
@@ -20,72 +20,70 @@
 #include "crypto/s2n_cipher.h"
 #include "crypto/s2n_fips.h"
 #include "crypto/s2n_openssl.h"
-
 #include "tls/s2n_crypto.h"
-
-#include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
+#include "utils/s2n_safety.h"
 
 /* LibreSSL and BoringSSL support the cipher, but the interface is different from Openssl's. We
  * should define a separate s2n_cipher struct for LibreSSL and BoringSSL.
  */
 #if !defined(LIBRESSL_VERSION_NUMBER) && !defined(OPENSSL_IS_BORINGSSL)
-/* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1
+    /* Symbols for AES-SHA1-CBC composite ciphers were added in Openssl 1.0.1
```

Comment:
Bleh, now this comment doesn't align with itself. Is there a fix for this?

Reply:
The fix is a manual indent.